docs/security/kinit.md

```diff
@@ -23,9 +23,13 @@
 
 # Kinit Auxiliary Service
 
-In order to work with a kerberos-enabled cluster, Kyuubi provides this kinit auxiliary service.
-It will periodically re-kinit with to keep the Ticket Cache fresh.
+Kinit auxiliary service is a critical service both for authentication between Kyuubi client/server 
+and for authentication between Kyuubi server/Hadoop cluster in a Kerberos environment.
+It will get a Kerberos Ticket Cache from KDC and periodically re-kinit to keep the Ticket Cache fresh.
 
+**Note**:
+Kinit auxiliary service is critical to Kyuubi Kerberos authentication, but not vice versa.
+Kinit auxiliary service can also work with other authentication mode.
```

Comment:
It seems more readable like this ?
```
- xxx
- xxx
```

otherwise the tow lines in md will be inlined.

Reply:
Thanks for the advice. Updated.